core-iconset.html

```diff
@@ -187,13 +202,13 @@
   },
 
   updateThemes: function () {
-    this.lightDom.querySelectorAll('property[theme]')
+    Polymer.dom.querySelectorAll('property[theme]', this)
       .forEach(function (themeProperty) {
         var offsetX = window.parseInt(
-          themeProperty.getAttribute('offsetx'), 10
+          themeProperty.getAttribute('offset-x'), 10
```

Comment:
Kinda funky, eh? This component expects its `offset-x` attribute to become an `offsetX` property, but its "theme" descendants leave `offset-x` alone and treat it as an attribute?